prowler

```diff
@@ -460,6 +477,8 @@ execute_check() {
   local caf_epic_var=CHECK_CAF_EPIC_$1
   CHECK_CAF_EPIC="${!caf_epic_var}"
 
+  local SUPPORTED_DB_PROVIDERS=$SUPPORTED_DB_PROVIDERS
```

Comment:
Why this? You are loading this variable from `include/db_connector`, am I right?